src/threads/openThread.spec.ts

```diff
@@ -0,0 +1,71 @@
+import * as parsers from '../parsers';
+import { handler } from './openThread';
+import { ForumTable } from './table';
+
+const mockEvent = (): any => ({});
+const mockContext = (): any => ({});
+
+describe('threads', () => {
+  const getTokenDataStub = jest.spyOn(parsers, 'extractTokenData');
+  const addThreadEventStub = jest.spyOn(ForumTable.prototype, 'addThreadEvent');
+  addThreadEventStub.mockReturnValue(Promise.resolve({} as any));
+
+  const tokenData = (n: number, rest?: Partial<parsers.TokenData>): parsers.TokenData => ({
+    participantId: `openThreadParticipantId-${n}`,
+    itemId: `openThreadItemId-${n}`,
+    userId: `openThreadUserId-${n}`,
+    isMine: true,
+    canWatchParticipant: true,
+    ...rest,
+  });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('open thread', () => {
+    it('should fail (401) when token data is invalid', async () => {
```

Comment:
"401" mentioned should probably be removed

Reply:
fixed